gui/gm-editor.lua

```diff
@@ -72,7 +85,7 @@ end
 
 GmEditorUi = defclass(GmEditorUi, widgets.Window)
 GmEditorUi.ATTRS{
-    frame=copyall(config.data),
+    frame=copyall(config.data.frame or {}),
```

Comment:
Instead of making `read_only` a global, you could add it here, then check it with `self.read_only` in other functions. It looks to me like it's only used from within this class (and when saving config)